lxd/network/driver_ovn.go

```diff
@@ -2545,9 +2545,49 @@ func (n *ovn) Start() error {
 	}
 
 	var projectID int64
+	var enableChassis int
 	err = n.state.Cluster.Transaction(func(tx *db.ClusterTx) error {
+		// Get the project ID.
 		projectID, err = tx.GetProjectID(n.project)
-		return err
+		if err != nil {
+			return err
+		}
+
+		// Get the node info.
```

Comment:
Please can this logic be moved into its own function. Something along the lines of `d.chassisEnabled(tx) (bool, error)`.
Then this can just be used as `if chassisEnabled {...`.

This has the benefit that the reader of `Start()` doesn't have to concern themselves as to the specific logic of chassis roles (unless they want to) and it doesn't expose the internal `enableChassis` variable that can be -1,0, or 1, but ultimately transforms into a bool.